### PR TITLE
UserResourceServiceImpl test suite 

### PR DESCRIPTION
**Pull request by JacobHarrisonQA**
title says it all

### CODE REVIEW

**Comment by JacobHarrisonQA** [comment]
go go

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImplTest.java [hunk 1]
```diff
@@ -2,23 +2,16 @@
...
+import org.mockito.InjectMocks;
 import uk.gov.ons.fwmt.job_service.rest.JobResourceService;
 import uk.gov.ons.fwmt.job_service.rest.dto.JobDto;
 
 import static org.junit.Assert.*;
 
-@RunWith(SpringRunner.class)
-@SpringBootTest(classes = {ApplicationConfig.class})
 @Ignore
 public class JobResourceServiceImplTest {
```
Comment (soundargit) [inline src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImplTest.java]: does this test run?

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java [hunk 2]
```diff
@@ -1,46 +1,188 @@
...
+      Boolean isActive = true;
+      UserDto expectedUserDto = new UserDto();
+      when(restTemplate.exchange(any(),any(),any(),eq(UserDto.class),eq(testAuthNo))).thenReturn(responseEntity);
+      when(responseEntity.getStatusCode()).thenReturn(HttpStatus.OK);
+      when(responseEntity.getBody()).thenReturn(expectedUserDto);
+
+      //When
+      Boolean result = userResourceService.existsByAuthNoAndActive(testAuthNo,isActive);
+
+      //Then
```
Comment (soundargit) [inline src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: this test should assert true

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java [hunk 3]
```diff
@@ -1,46 +1,188 @@
...
+    UserDto expectedUserDto = new UserDto();
+    when(restTemplate.exchange(any(),any(),any(),eq(UserDto.class),eq(testAlthNo))).thenReturn(responseEntity);
+    when(responseEntity.getStatusCode()).thenReturn(HttpStatus.OK);
+    when(responseEntity.getBody()).thenReturn(expectedUserDto);
+
+    //When
+    Boolean result = userResourceService.existsByAlternateAuthNoAndActive(testAlthNo,isActive);
+
+    //Then
+    assertFalse(result);
```
Comment (soundargit) [inline src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: should assert true

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImplTest.java (hunk 1 above)
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImplTest.java]: That class im testing next and will be changes to correct layout

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java (hunk 2 above)
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: It passes currently and fails if assertTrue

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java (hunk 3 above)
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: It passes currently and fails if assertTrue

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java (hunk 2 above)
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: fixed

File: src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java (hunk 3 above)
Comment (JacobHarrisonQA) [reply src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/UserResourceServiceImplTest.java]: fixed

**Pull request merged**